src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -762,9 +795,9 @@ static int glb_pick(grpc_exec_ctx *exec_ctx, grpc_lb_policy *pol,
       GRPC_LB_POLICY_UNREF(exec_ctx, glb_policy->wc_arg.rr_policy, "glb_pick");
 
       /* add the load reporting initial metadata */
-      initial_metadata_add_lb_token(pick_args->initial_metadata,
-                                    pick_args->lb_token_mdelem_storage,
-                                    glb_policy->wc_arg.lb_token);
+      initial_metadata_add_lb_token(
+          pick_args->initial_metadata, pick_args->lb_token_mdelem_storage,
+          user_data_copy(glb_policy->wc_arg.lb_token));
```

Comment:
it's also used on line 188 (https://github.com/grpc/grpc/pull/7746/files/6cc44fcf3ab68d55eadec5590e35b9268cc2d6a6..35c2aba849e0f6852e3538fc488a6a2afec81c09#diff-d2af040e96ac2ed8c608b6624293f9d1R188)